src/test/CMakeLists.txt

```diff
@@ -307,6 +307,44 @@ target_link_libraries(ceph_test_librgw_file_cd
   )
 install(TARGETS ceph_test_librgw_file_cd DESTINATION ${CMAKE_INSTALL_BINDIR})
 
+if(WITH_RADOSGW_D4N)
```

Comment:
I think these changes and the subsequent tests should go into `src/test/rgw`. I know there are some rgw related tests in this dir and I'm not sure why that is.